cmd/aws.go

```diff
@@ -26,18 +23,9 @@ var (
 	accessKeyID     string
 	secretAccessKey string
 	region          string
-	logger          *micrologger.MicroLogger
```

Comment:
The logger is now in the `root.go`. I like the more structured project layout in `opsctl` more than this here but I am not going to rewrite everything now.